src/libraries/System.Runtime.Serialization.BinaryFormat/src/System/Runtime/Serialization/BinaryFormat/BinaryType.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.Serialization.BinaryFormat;
+
+/// <summary>
+/// Identifies the remoting type of a class member or array item.
+/// </summary>
+/// <remarks>
+/// BinaryType enumeration is described in <see href="https://learn.microsoft.com/openspecs/windows_protocols/ms-nrbf/054e5c58-be21-4c86-b1c3-f6d3ce17ec72">[MS-NRBF] 2.1.2.2</see>.
+/// </remarks>
+internal enum BinaryType : byte
+{
+    /// <summary>
+    ///  Type is defined by <see cref="PrimitiveType"/> and it is not a string.
```

Comment:
```suggestion
    ///  The type is defined by <see cref="PrimitiveType"/> and it is not a string.
```

(similarly for the rest)